src/modules/endpoint/__tests__/Endpoint.spec.js

```diff
@@ -89,7 +89,7 @@ describe('Endpoint', () => {
 
     beforeEach(() => {
       clock = lolex.install({now: mockedTime, toFake: []})
-      prevError = console.error
+      prevError = console.error // todo: Find better way of allowing console.error
```

Comment:
For now this should be enough. We should keep an eye on this if we really want that or if there is a better way.